### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ccorneliusHsv**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
 import axios from 'axios';
 import './App.css';
-import Endpoint from './components/Endpoint';
-import Mode from './components/Mode';
-import Panel from "./components/Panel";
-import Menu from "./components/Menu";
-import MenuIcon from "./components/MenuIcon";
-import Split from "./components/Split";
-import ActionButton from "./components/ActionButton";
+import {Endpoint, Mode, Panel, Menu, MenuIcon, Split, ActionButton} from './components/index.js';
```
Comment (jkearle) [inline src/App.js]: Probably should remove "Menu" from the import if we're not using it. I get compile warnings about it not being used.

File: src/App.js [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
     constructor(props) {
         super(props);
 
         this.state = {
             menu: false,
             split: false,
             panel: false,
-            endpoint: "",
-            query1: "",
+            endpoint: `http://10.12.51.25:9200/offers/offer/_search`,
```
Comment (jkearle) [inline src/App.js]: probably need to remove these default state settings for endpoint and query1

File: src/components/Endpoint.js [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
     render() {
         return (
             <div className="Endpoint Component">
-                <label htmlFor="endpoint">Search Endpoint</label>
+                <div className={'labelsDiv'}>
+                    <div className={'endpointLabelDiv'}>
+                        <label htmlFor="endpoint">Search Endpoint</label>
+                    </div>
+                    <div className={'statusLabelDiv'}>
+                        <label>{'Status:' + this.props.statusText}</label>
```
Comment (jkearle) [inline src/components/Endpoint.js:13]: Let's talk about this. I see why you put this here, but it seems like a separate concern from the rest of the endpoint component

File: src/components/Stats.js [hunk 4]
```diff
@@ -3,17 +3,19 @@ import './Stats.css';
 
 class Stats extends Component {
     render() {
-        let took = "n/a";
+        const {took} = this.props;
```
Comment (jkearle) [inline src/components/Stats.js:6]: why was this added? don't see it used anywhere

File: src/App.js (hunk 1 above)
Comment (ccorneliusHsv) [reply src/App.js]: Removed Menu and Menu Icon, as well as unused methods.

File: src/components/Endpoint.js (hunk 3 above)
Comment (ccorneliusHsv) [reply src/components/Endpoint.js:13]: Per our discussion we are going to address this in a future ticket.

File: src/App.js (hunk 2 above)
Comment (ccorneliusHsv) [reply src/App.js]: Done.

File: src/components/Stats.js (hunk 4 above)
Comment (ccorneliusHsv) [reply src/components/Stats.js:6]: Yeah, that was a step to make a deconstructed variable, that I didn't finish.
